### PR TITLE
feat: device account config v3

### PR DESCRIPTION
**Pull request by ntheile**

### CODE REVIEW

**Comment by ntheile** [comment]
config for https://github.com/GaloyMoney/galoy/pull/2747

File: charts/galoy/values.yaml [hunk 1]
```diff
@@ -668,6 +670,31 @@ oathkeeper:
           "authorizer": { "handler": "allow" },
           "mutators": [{ "handler": "noop" }]
         },
+        {
+          "id": "device-login-routes",
+          "upstream": {
+            "url": "http://api:4002"
+          },
+          "match": {
+            "url": "<(http|https)>://<[a-zA-Z0-9-.:]+>/auth/<(create/device-account|login/device-account)>",
```
Comment (nicolasburtey) [inline charts/galoy/values.yaml:679]: need to be narrow down to the proper link

**Review by nicolasburtey — COMMENTED** [review]
expect https://github.com/GaloyMoney/charts/pull/3249/files#r1228648555 LGTM

**Review by UncleSamtoshi — APPROVED** [review]

File: charts/galoy/values.yaml [hunk 2]
```diff
@@ -830,6 +849,8 @@ kratos:
   secret:
     enabled: true
     nameOverride: ""
+  image:
+    tag: v0.13.0
```
Comment (ntheile) [inline charts/galoy/values.yaml:853]: @krtk6160 

i had to add this kratos image tag back in. In my local dev charts environment the kratos container would not boot up properly and was trying to use by default an older version of kratos 0.11.0 that does not work with out current backend codebase. any ideas?

Comment (krtk6160) [reply charts/galoy/values.yaml:853]: @ntheile It's possible you have old charts in your `charts/charts/galoy/charts` directory. Delete all charts there, then run `helm dep up` in the `charts/charts/galoy` directory and retry.

Comment (ntheile) [reply charts/galoy/values.yaml:853]: it worked thanks!

**Pull request merged**